src/store/rootSaga.js

```diff
@@ -0,0 +1,12 @@
+import { all } from 'redux-saga/effects'
+
+/**
+ * Generator funktion ska alltid ha * lär dig använda
+ */
+
+export function* rootSaga(){
+  yield all([
+      //
```

Comment:
Du måste länka in din saga här. Annars kommer den inte fungera.

Reply:
Sagan är nu länkad.